person.rb

```diff
@@ -1,6 +1,13 @@
-class Person
-  def initialize(age, name = 'unknown', parent_permission: true)
-    @id = Random.rand(1..1000)
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
```

Comment:
**[Optional]**

- [ ] You have successfully implemented the error exception in your correct_name method but it would be nice if you make you method as below :arrow_down: to make your error message more professional and meaningful :+1:

```ruby
    def correct_name
        raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
    end
```